contents/csharp/Xamarin.md

```diff
@@ -0,0 +1,144 @@
+<frontmatter>
+  title: Xamarin
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Xamarin
+
+**Author: [Ni Tianzhen](https://github.com/niqiukun)**
+
+<box id="article-toc">
+
+* [Introduction to Xamarin](#introduction-to-xamarin)
+    * [What is Xamarin‎](#what-is-xamarin)
+    * [How Xamarin works](#how-xamarin-works)
```

Comment:
The table of contents can start from `what is Xamarin` instead of `Introduction to Xamarin`